tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Check that password for keystore is defined
+  assert:
+    that: corda_password_keystore is defined
+    msg: "Having password to keystore in config file is not idea, but for now please set 'corda_password_keystore' to a value"
```

Comment:
I believe there is a typo in `Having password to keystore in config file is not idea`

Also, even the source of `corda_password_keystore` can be anything, an environment variable, a variable from vault, and so on.